tests.py

```diff
@@ -226,15 +234,15 @@ def test_eq_value_error(self):
 
     def test_iter_items(self):
         items = [(k, v) for k, v in self.value.iteritems()]
-        self.assertEqual(self.value.items(), items)
+        self.assertCountEqual(self.value.items(), items)
```

Comment:
This isn't counting that the items are the same, I'd prefer to not lose the equality checks. Can't you just sort both here?

Reply:
I really have the same question/request for all of the tests that were changed from equality checks to count checks.

Reply:
assertCountEqual check if they have the same elements in the same number.

Reply:
Now that we dropped 3.4 all test changes are there because python 3.5 can't guarantee insertion order.

Reply:
TIL re assertCountEqual